package.json

```diff
@@ -58,16 +56,17 @@
     "enzyme-adapter-preact-pure": "^4.0.1",
     "escape-html": "^1.0.3",
     "escape-string-regexp": "^4.0.0",
-    "eslint": "^8.3.0",
-    "eslint-config-hypothesis": "^2.6.0",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-mocha": "^10.0.1",
-    "eslint-plugin-react": "^7.12.4",
-    "eslint-plugin-react-hooks": "^4.0.4",
+    "eslint": "^9.12.0",
+    "eslint-config-hypothesis": "file:.yalc/eslint-config-hypothesis",
```

Comment:
A `.yalc` reference crept into the package.json file.

Reply:
Yeah, I'll change that once `eslint-config-hypothesis` v3 is released with support for flat config.